mariadb/config.yaml

```
@@ -10,8 +10,8 @@ arch:
   - aarch64
   - amd64
   - i386
-backup_post: s6-svc -wd -d /run/s6/legacy-services/mariadb-lock/
-backup_pre: s6-svc -wU -u /run/s6/legacy-services/mariadb-lock/
+backup_post: /command/s6-rc -d change mariadb-lock
+backup_pre: /command/s6-rc -u change mariadb-lock
```

Shouldn't this be

```suggestion
backup_post: /command/s6-rc -d change mariadb-lock-post
backup_pre: /command/s6-rc -u change mariadb-lock-post
```

Or is that some s6 naming magic?

---

Yeah, magic. I've spent a few hours figuring out how to do this. This is a working solution, seems to be right, but I'm not a professional developer, so I don't have enough experience to say, that "this is the correct solution". :/

So, `mariadb-lock` is a bundle, that consist of longrun `mariadb-lock-core` and oneshot `mariadb-lock-post`, we `up` and `down` the bundle and S6 manages the details. The `mariadb-lock` bundle is not part of the `user` bundle, so it is not started when the add-on starts.

I've experimented with a single service, but there were always some problems, this longrun+oneshot combination seems to be right.

---

I see and it seems to work well here:

```
[11:24:05] INFO: Start MariaDB client (to lock tables for backups)
[11:24:05] INFO: MariaDB tables locked
[11:24:07] INFO: MariaDB tables unlocked
[11:24:07] INFO: MariaDB client (to lock tables for backups) stopped
```

Do you happen to have a pointer where this bundle feature is documented?